misc/ballerina-bindgen/src/main/java/org/ballerinalang/bindgen/utils/BindgenNodeFactory.java

```diff
@@ -382,7 +381,7 @@ private static ReturnTypeDescriptorNode getExternalFunctionSignatureReturnType(B
     }
 
     private static boolean isMultiDimensionalArray(String className) {
-        return className.lastIndexOf('[') > 0;
+        return className.codePoints().filter(ch -> ch == '[').count() > 1;
```

Comment:
Can we use `anyMatch` here ?

Reply:
Here the logic is to check if the count is greater than 1, therefore, it would be difficult to use `anyMatch` here.